<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutCaptureDialogBase</class>
 <widget class="QDialog" name="ShortcutCaptureDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Shortcut Sequence</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="descrLabel">
     <property name="toolTip">
      <string>To enter a shortcut, choose a key sequence, for example, Ctrl+A, and press it on your keyboard. 
A key sequence can be implemented into another: Ctrl+B, and then Ctrl+C is a valid shortcut.
Up to four combinations can be used: Ctrl+A then Ctrl+B then Ctrl+C then Ctrl+D is also a valid shortcut.
Note: this last shortcut would be noted &quot;Ctrl+A,Ctrl+B,Ctrl+C,Ctrl+D&quot;</string>
     </property>
     <property name="text">
      <string>Press up to four key combinations to enter shortcut sequence.
Note: &quot;Ctrl+Shift+1&quot; is one key combination.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="oshrtTextLabel">
     <property name="text">
      <string>Old shortcuts:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="oshrtLabel">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="accessibleName">
      <string>Old shortcuts</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="nshrtTextLabel">
     <property name="text">
      <string>New shortcut:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="nshrtLabel">
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="accessibleName">
      <string>New shortcut</string>
     </property>
     <property name="accessibleDescription">
      <string>Type to set shortcut</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string comment="placeholder text">Type to set shortcut</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="accessibleDescription">
        <string>Clear new shortcut.</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="accessibleDescription">
        <string>Add the new shortcut to the list of existing shortcuts.</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="accessibleDescription">
        <string>Replace the old shortcut(s) with new one.</string>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ShortcutCaptureDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
